### DISCUSSION TITLE
Set a path to virtual env(no general env) when convert to exe

### DISCUSSION THREAD

**Discussion by PawelKinczyk** [brentvollebregt/auto-py-to-exe#385] [Q&A] [answered]
Hi, i try to convert my simple program. I have all packages in virtual env but not in main env. Because of this i had problem with error: ModuleNotFoundError: No module named 'docxtpl' etc. I worked around this and install all packages in my main env. Now all works good but there is a option to set other path for virtual env in "Auto Py to Exe" GUI? Thanks in advance!

**Comment by brentvollebregt** [comment, marked as answer]
auto-py-to-exe will get the modules from the current environment - so if you installed auto-py-to-exe in the main env and then your dependencies in a virtual env, auto-py-to-exe cannot see the dependencies in the virtual env.

My recommendation is to install auto-py-to-exe in the virtual env and validate you are running the correct one by using `where auto-py-to-exe` - the top row should point to inside your virtual env folder.

**Reply by PawelKinczyk** [reply]
Thank you so much for your time. All works perfect!
